b2sdk/file_version.py

```diff
@@ -412,8 +465,8 @@ def __init__(
         self.content_disposition = content_disposition
         self.content_length = content_length
         self.content_language = content_language
-        self._expires = expires  # TODO: parse the string representation of this timestamp to datetime in DownloadVersionFactory
-        self._cache_control = cache_control  # TODO: parse the string representation of this mapping to dict in DownloadVersionFactory
+        self.expires = expires
+        self.cache_control = cache_control
```

Comment:
`expires` returned by Backblaze is not guaranteed to be a valid HTTP-date as defined in standards. We need to preserve the raw output anyways. So "parsing the string representation" is not going to happen 100%. Instead, I keep `self.expires` as a raw string and do the optional parsing with `self.expires_parsed()`.